code/pagetypes/Forum.php

```diff
@@ -109,7 +109,7 @@ function canModerate($member = null) {
 		if(!$member) return false;
 		
 		// Admins
-		if ($this->canEdit($member)) return true; 
+		if ($member->inGroup('administrators')) return true; 
```

Comment:
I'll add the later in a new PR on 0.8, this is a patch level bug fix for the 0.7.